src/tools/default_flags_table/reco_flags.py

```diff
@@ -74,7 +74,100 @@
+    ('BEMC:EcalBarrelScFiRecHits:pedestalMean',                 '400',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:pedestalSigma',                '3.2',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:resolutionTDC',                '10*picosecond',                   ''),
+    ('BEMC:EcalBarrelScFiRecHits:thresholdFactor',              '5.0',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:thresholdValue',               '0.0',                             ''),
+    ('BEMC:EcalBarrelScFiRecHits:samplingFraction',             '0.10262666247845109',             ''),
+
+    ('BEMC:EcalBarrelscFiMergedHits:input_tag',                 'EcalBarrelScFiRecHits',           ''),
+    ('BEMC:EcalBarrelscFiMergedHits:fields',                    'fiber',                           ''),
+    ('BEMC:EcalBarrelscFiMergedHits:refs',                      '1',                               ''),
+
+    ('BEMC:EcalBarrelImagingProtoClusters:input_tag',           'EcalBarrelImagingRecHits',        'Name of input collection to use'),
+    ('BEMC:EcalBarrelImagingProtoClusters::localDistXY',        '2.0*mm,2*mm',                     ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::layerDistEtaPhi',    '10*mrad,10*mrad',                 ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::sectorDist',         '3.0*cm',                          ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::minClusterHitEdep',  '0.',                              ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::minClusterCenterEdep', '0.',                            ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::minClusterEdep',     '0.5*MeV',                         ''),
+    ('BEMC:EcalBarrelImagingProtoClusters::neighbourLayersRange', '2.0',                           ''),
+
+    ('BEMC:EcalBarrelScFiProtoClusters:input_tag',              'EcalBarrelScFiMergedHits', 'Name of input collection to use'),
+    ('BEMC:EcalBarrelScFiProtoClusters:splitCluster',           'false',                           ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:minClusterHitEdep',      '1.0*MeV',                         ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:minClusterCenterEdep',   '10.0*MeV',                        ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:sectorDist',             '5.0*cm',                          ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:localDistXY',            '',                                ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:localDistXZ',            '',                                ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:localDistYZ',            '',                                ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:globalDistRPhi',         '',                                ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:globalDistEtaPhi',       '',                                ''),
+    ('BEMC:EcalBarrelScFiProtoClusters:dimScaledLocalDistXY',   '30*mm,30*mm',                     ''),
```

Comment:
This is the default value in https://github.com/eic/EICrecon/blob/e6f1f93aba923c36939f63581dbe8df9fb04e3ba/src/algorithms/calorimetry/CalorimeterIslandCluster.cc#L92 which gets ignored if any other distance parameter is defined (in our case should be in XZ). So instead of passing 30 mm, 30 mm here: https://github.com/eic/EICrecon/blob/e6f1f93aba923c36939f63581dbe8df9fb04e3ba/src/detectors/BEMC/ProtoCluster_factory_EcalBarrelScFiProtoClusters.h#L42 the `localDistXZ` should be set like here: https://eicweb.phy.anl.gov/EIC/benchmarks/physics_benchmarks/-/blob/master/options/reconstruction.py#L655.

Reply:
OK. Removed the setting of u_dimScaledLocalDistXY so now only localDistXZ is set.